src/internal/Popover.js

```diff
@@ -310,6 +310,8 @@ export default class Popover extends Component {
     };
   }
 
+  handleGetOffsetTop = getOffsetTop;
```

Comment:
What about exporting the two functions instead of attaching them to the instance of the component?
I feel like it will be more explicit.